sci-geosciences/grass/grass-8.4.0.ebuild

```diff
@@ -0,0 +1,277 @@
+else
+	MY_P="${P/_rc/RC}"
+	SRC_URI="https://grass.osgeo.org/${MY_PM}/source/${MY_P}.tar.gz"
+	if [[ ${PV} != *_rc* ]] ; then
+		KEYWORDS="~amd64 ~ppc ~x86"
+	fi
+
+	S="${WORKDIR}/${MY_P}"
+fi
+
+IUSE="blas bzip2 cxx fftw geos lapack las mysql netcdf nls odbc opencl opengl openmp pdal png postgres readline sqlite svm threads tiff truetype X zstd"
+REQUIRED_USE="
+	${PYTHON_REQUIRED_USE}
+	opengl? ( X )"
+
+RDEPEND="
+	${PYTHON_DEPS}
+	>=app-admin/eselect-1.2
+	$(python_gen_cond_dep '
+		dev-python/numpy[${PYTHON_USEDEP}]
+		dev-python/ply[${PYTHON_USEDEP}]
+		dev-python/python-dateutil[${PYTHON_USEDEP}]
+	')
+	sci-libs/gdal:=
+	sys-libs/gdbm:=
+	sys-libs/ncurses:=
+	sci-libs/proj:=
+	sys-libs/zlib
+	media-libs/libglvnd
+	media-libs/glu
+	blas? (
+		virtual/cblas[eselect-ldso(+)]
+		virtual/blas[eselect-ldso(+)]
+	)
+	bzip2? ( app-arch/bzip2:= )
+	fftw? ( sci-libs/fftw:3.0= )
+	geos? ( sci-libs/geos:= )
+	lapack? ( virtual/lapack[eselect-ldso(+)] )
+	las? ( sci-geosciences/liblas )
+	mysql? ( dev-db/mysql-connector-c:= )
+	netcdf? ( sci-libs/netcdf:= )
+	odbc? ( dev-db/unixODBC )
+	opencl? ( virtual/opencl )
+	opengl? ( virtual/opengl )
+	pdal? ( >=sci-libs/pdal-2.0.0:= )
+	png? ( media-libs/libpng:= )
+	postgres? ( >=dev-db/postgresql-8.4:= )
+	readline? ( sys-libs/readline:= )
+	sqlite? ( dev-db/sqlite:3 )
+	svm? ( sci-libs/libsvm:= )
+	tiff? ( media-libs/tiff:= )
+	truetype? ( media-libs/freetype:2 )
+	X? (
+		$(python_gen_cond_dep '
+			>=dev-python/matplotlib-1.2[wxwidgets,${PYTHON_USEDEP}]
+			dev-python/pillow[${PYTHON_USEDEP}]
+			>=dev-python/wxpython-4.1:4.0[${PYTHON_USEDEP}]
+		')
+		x11-libs/cairo[X]
+		x11-libs/libICE
+		x11-libs/libSM
+		x11-libs/libX11
+		x11-libs/libXext
+		x11-libs/libXt
+	)
+	zstd? ( app-arch/zstd:= )"
+DEPEND="${RDEPEND}
+	X? ( x11-base/xorg-proto )"
+BDEPEND="
+	app-alternatives/yacc
+	app-alternatives/lex
+	sys-devel/gettext
+	virtual/pkgconfig
+	X? ( dev-lang/swig )"
+
+PATCHES=(
+	# bug 746590
+	"${FILESDIR}/${PN}-flock.patch"
+)
+
+pkg_setup() {
+	if use lapack; then
+		local mylapack=$(eselect lapack show)
+		if [[ -z "${mylapack/.*reference.*/}" ]] && \
+			[[ -z "${mylapack/.*atlas.*/}" ]]; then
+			ewarn "You need to set lapack to atlas or reference. Do:"
+			ewarn "   eselect lapack set <impl>"
+			ewarn "where <impl> is atlas, threaded-atlas or reference"
+			die "setup failed"
+		fi
+	fi
+
+	if use blas; then
+		local myblas=$(eselect blas show)
+		if [[ -z "${myblas/.*reference.*/}" ]] && \
+			[[ -z "${myblas/.*atlas.*/}" ]]; then
+			ewarn "You need to set blas to atlas or reference. Do:"
+			ewarn "   eselect blas set <impl>"
+			ewarn "where <impl> is atlas, threaded-atlas or reference"
+			die "setup failed"
+		fi
+	fi
+
+	python-single-r1_pkg_setup
+}
+
+src_prepare() {
+	# Fix unversioned python calls
+	sed -e "s:=python3:=${EPYTHON}:" -i "${S}/lib/init/grass.sh" || die
+	sed -e "s:= python3:= ${EPYTHON}:" -i "${S}/include/Make/Platform.make.in" || die
```

Comment:
```
* QA Notice: addpredict called with colon-separated argument
```

Reply:
That's in https://github.com/gentoo/gentoo/blob/master/sci-geosciences/grass/grass-8.4.0.ebuild#L151-L162 and should really be moved elsewhere.